### PR TITLE
refactor: Settings simplified

### PR DESCRIPTION
**Pull request by Ambrdctr**

### CODE REVIEW

**Review by adrien-coye — COMMENTED** [review]
LGTM so far

File: Mail/Views/Settings/SettingsSectionHeaderView.swift [hunk 1]
```diff
@@ -0,0 +1,48 @@
+//
```
Comment (valentinperignon) [inline Mail/Views/Settings/SettingsSectionHeaderView.swift:1]: Aïe Aïe Aïe

File: Mail/Views/Settings/SettingsSectionHeaderView.swift [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import SwiftUI
+
+struct SettingsSectionHeaderView: View {
+    let title: String
+    let separator: Bool
+
+    init(title: String, separator: Bool = false) {
```
Comment (valentinperignon) [inline Mail/Views/Settings/SettingsSectionHeaderView.swift:26]: Do you prefer an `init()` over `var separator = false ?`

File: Mail/Views/Settings/General/SettingsListView.swift [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import MailCore
+import MailResources
+import SwiftUI
+
+struct SettingsListView: View {
+    @State private var lockEnable = false
```
Comment (PhilippeWeidmann) [inline Mail/Views/Settings/General/SettingsListView.swift:24]: Unused property

**Review by valentinperignon — APPROVED** [review]

**Review by PhilippeWeidmann — APPROVED** [review]

**Pull request merged**